app/models/message.rb

```diff
@@ -2,15 +2,14 @@ class Message < ApplicationRecord
   has_many :reports, :through => :logs
   has_many :logs
   validates_lengths_from_database
-  validates :value, :digest, :presence => true
```

Comment:
is dropping the `value` validation on purpose?

Reply:
Yes, I recall I was having some issues with tests. Let me remove this to see if they are passing.

Reply:
It had something to do with the database index, presense validator requires an unique index and hash index indeed is exactly (only) unique.